milestone-3/frontend/src/components/recipePage.tsx

```diff
@@ -0,0 +1,71 @@
+import  { ChangeEvent, useState } from 'react';
+import { Link } from "react-router-dom";
+
+interface Recipe {
+    name: string;
+    image: string;
+    description: string;
+    ingredients: string [];
+    instructions: string [];
+}
```

Comment:
you can replace this with an import from your types file, removes duplicate code.